wiki/Skinning/Interface/en.md

```diff
@@ -952,7 +951,7 @@ _`fail-background.png`_ (transparent; not included)
 
 **Note:**
 
-- This element appears when game-play is paused or failed.
+- This element appears when a map is paused or failed.
```

Comment:
`a map` could mean the `.osk` file, `.osu` file, editing in the editor... 👎 

(will suppress this for previous and next instances)